style.css

```diff
@@ -45,12 +48,15 @@ body {
 
 a {
     text-decoration: none;
-    color: red
+    color: red;
```

Comment:
dont give directly color like red blue orange etc try to give in hex code or rgba